biscuit-pulsar/src/main/java/com/clevercloud/biscuitpulsar/AuthenticationProviderBiscuit.java

```diff
@@ -117,6 +123,10 @@ private String parseBiscuit(final String biscuit) throws AuthenticationException
     }
   }
 
+  private boolean isJWT(String jwt) {
+    return jwt.split("\\.").length == 3;
```

Comment:
maybe check with the spec that there will always be three dots? https://tools.ietf.org/html/rfc7519#section-7.2

Reply:
I bumped in 3b8944f don't think we can go further without jwt keys, WDYT?